cse/indexing/FileIndexer.py

```diff
@@ -70,25 +69,34 @@ def __processComment(self, cid, comment):
         tokenPositionsDict = {}
         for token, position in tokenTuples:
             positionList = tokenPositionsDict.get(token, [])
-            positionList.append(position)
-            positionList.sort()
+            positionList.append(int(position))
```

Comment:
Is `tokenTuples` sorted?and is it a ordered list?

Reply:
Every python list is sorted. It will be ordered by `<` in line [77](https://github.com/CodeLionX/CommentSearchEngine/pull/49/files/6ac37c65b3f0b0e29eee50d1993a3d103ceb7eb9#diff-eecd07539f2dcc11f2c9c420a45c7069R77) before it is passed to the index.

Reply:
ok